### ISSUE TITLE
Build a terraform script that will create an EC2 instance on AWS using a free tier account.

### ISSUE THREAD

**Issue by Popara5** [QuestLogicSolutions/qlslearning-project1-terraformEC2#1]

**Comment by Popara5** [comment]
Hi Franck, can I use an existing security group ID on AWS or create a new one?

**Comment by Balibirba23** [comment]
Hi ok cool

**Comment by cloudcwfranck** [comment]
@Popara5  Yes as long as it fit the requirements, and that you can ssh at the end of the project.

**Comment by Balibirba23** [comment]
Ok, thanks.

**Comment by YoungSwiftling** [comment]
Test comment!

  223  terraform version
  224  terraform init
  225  pwd
  226  cd 
  227  pwd
  228  ll
  229  ls
  230  cd TerraForm
  231  ls
  232  terraform init
  233  terraform plan
  234  terraform init
  235  terraform plan
  236  terraform init
  237  terraform plan
  238  terraform apply

**Comment by Popara5** [comment]
I have successfully completed testing terraform by creating a single aws ec2. 

commands I used ,
- pwd 
- cd Project-1
- terraform init
- terraform plan 
- terraform apply
